From 6a355cad49b86800c15d88f0be84afa62ff338d3 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 31 Mar 2020 12:24:53 +0200
Subject: [PATCH 2/2] bootcmd: Flash only if bootswitch in EXT position

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8mm_var_dart.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 86424e4..a307ea8 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -195,6 +195,7 @@
 
 #define CONFIG_BOOTCOMMAND \
 	"run ramsize_check; " \
+	"if bootswitchpos EXT; then setenv resin_flasher_skip 0; else setenv resin_flasher_skip 1; fi; " \
 	"setenv resin_kernel_load_addr ${loadaddr};" \
 	"run resin_set_kernel_root; run set_os_cmdline; gpio set 137; " \
 	"setenv mmcdev ${resin_dev_index};" \
-- 
2.7.4

